code/components/jomjol_image_proc/CRotateImage.cpp

```diff
@@ -336,7 +339,7 @@ void CRotateImage::Translate(int _dx, int _dy)
     memCopy(odata, rgb_image, memsize);
     if (!ImageTMP)
     {
-        stbi_image_free(odata);
```

Comment:
Have you checked if in the function stbi_image_free something is handled special that you can repalce it?